### DISCUSSION TITLE
Report this project and delete it

### DISCUSSION THREAD

**Discussion by Rubelz** [hacksider/Deep-Live-Cam#330] [General]
It might lead to bad things, stop it, please, no judges

**Comment by anhdungadg** [comment]
I think this project is very suitable for research and study. A 🔪 in the hands of a murderer becomes a dangerous weapon, but in the hands of a chef, it creates masterpieces.

**Reply by dablenparty** [reply]
Your analogy is comparing apples to oranges. The knife has a reason to exist; why does this exist? What kind of _ethical_ research does this contribute to? The way I see it, this repo is a perfect example of “just because you can doesn’t mean you should.”
